src/account/isValidPassword.js

```diff
@@ -0,0 +1,18 @@
+// Atleast one digit
+const oneDigit = /^(?=.*\d)/;
+
+// Atleast one uppercase character
+const oneUpperCase = /^(?=.*[A-Z])/;
+
+// Atleast one special character
+// Using this regex to match all non-alphanumeric characters to match server-side implementation
+// https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/user/SpecialCharacterValidationRule.java#L39
+const oneSpecialCharacter = /[^a-zA-Z0-9]/;
+
+export default function isValidPassword(value) {
+    if (!value) {
+        return true;
```

Comment:
Absence of a password is a valid password? I'd prefer if the App didn't call `isValidPassword` if it had no value to call it with.